### ISSUE TITLE
MultiBox loss goes to infinity

### ISSUE THREAD

**Issue by RogerAylagas** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#43]
Hi,
I hope you can help me to solve the following issue.
While training, in the first steps (first half of the batch) error was being computed well, but then suddenly the MultiBox error turned to infinity (I get inf)
Does anyone know what is the problem?
Thank you so much

**Comment by AntoineMath** [comment]
Same problem here, I'm using this code for a long time and this is the first time it happens. 
I am using a depth and a thermal image dataset. 
No problem with the thermal dataset, but when I train on the depth dataset, i get (inf) for the validation loss : 
![validatation_loss_inf](https://user-images.githubusercontent.com/44801785/76987168-053f9600-6943-11ea-9577-07eeddbfdde1.png)

**Comment by RogerAylagas** [comment]
In my case it is the confidence loss what goes to inf. Is it the same for you @AntoineMath ?

**Comment by AntoineMath** [comment]
@RogerAylagas For me it's the location loss. But it's strange because it's around 0.9 and 1.2 and suddenly I got an "inf" for the loc loss (right tensor). Maybe I should inspect the gradient.
![conf_loc_loss](https://user-images.githubusercontent.com/44801785/77096561-cc212780-6a0f-11ea-8383-5d4e0373b17c.png)

**Comment by RogerAylagas** [comment]
What learning rate are you using @AntoineMath ?

**Comment by abhisingh977** [comment]
Hey, guys,
Are you using the same dataset?
If not then this is happening because the lr is high for your dataset. Reduce the lr. 
As soon as the loss reaches more the thousand it starts showing nan or inf.
